test/mox_test.exs

```diff
@@ -392,40 +474,96 @@ defmodule MoxTest do
       set_mox_private()
 
       verify!(CalcMock)
+      verify!(SciCalcOnlyMock)
       expect(CalcMock, :add, fn x, y -> x + y end)
+      expect(SciCalcOnlyMock, :exponent, fn x, y -> x * y end)
 
       message = ~r"expected CalcMock.add/2 to be invoked once but it was invoked 0 times"
-      assert_raise Mox.VerificationError, message, &verify!/0
+      assert_raise Mox.VerificationError, message, fn -> verify!(CalcMock) end
+      message = ~r"expected SciCalcOnlyMock.exponent/2 to be invoked once but it was invoked 0 times"
+      assert_raise Mox.VerificationError, message, fn -> verify!(SciCalcOnlyMock) end
```

Comment:
**Annotation:** Even though these existing tests are in the `describe "verify!/1"` section, they used to be calling the `verify!/0` method! With this change now we're actually :one: calling the `verify!/1` method and :two: asserting that the behavior changes appropriately when there are multiple mocks used within the same test case.